Description: make build reproducible
 Stops build hostname and directory from being recorded in the artifact.
Author: Sascha Steinbiss <sascha@steinbiss.name>
--- a/source/Makefile
+++ b/source/Makefile
@@ -18,7 +18,7 @@
 LDFLAGS_Mac_static :=-pthread -lz -static-libgcc htslib/libhts.a
 LDFLAGS_gdb += $(LDFLAGS_shared)
 
-COMPTIMEPLACE := -D'COMPILATION_TIME_PLACE="$(shell echo `date` $(HOSTNAME):`pwd`)"'
+COMPTIMEPLACE := -D'COMPILATION_TIME_PLACE="<unrecorded>"'
 
 CXXFLAGS_common := -pipe -std=c++11 -Wall -Wextra -fopenmp $(COMPTIMEPLACE) $(CCFLAGS_common_add)
 CXXFLAGS_main += -O3 $(CXXFLAGS_common)
